### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by tkdchen**
Most of the changes were done after testing MTS locally.

### CODE REVIEW

File: message_tagging_service/tagging_service.py [hunk 1]
```diff
@@ -324,10 +322,16 @@ def handle(rule_defs, event_msg):
...
 
-    rule_matches = list(filter(truth, (
-        RuleDef(rule_def).match(modulemd)
-        for rule_def in rule_defs
-    )))
+    rule_matches = []
+    for i, rule_def in enumerate(rule_defs, 1):
+        rd = RuleDef(rule_def)
+        logger.info('[%s] Checking rule definition: %s', i, rd.id)
+        match = rd.match(modulemd)
```
Comment (lcarva) [inline message_tagging_service/tagging_service.py]: You could throw the log statement in `RuleDef.match` so you don't have to rewrite this bit of code.

File: message_tagging_service/__init__.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 
 import logging
 
+from message_tagging_service.mts_config import mts_conf
+
 logging.basicConfig(
     format='%(asctime)s- %(name)s - %(levelname)s - %(message)s',
-    level=logging.DEBUG)
+    level=getattr(logging, mts_conf.log_level, 'INFO'))
```
Comment (lcarva) [inline message_tagging_service/__init__.py]: Is `getattr` needed here? It's defined in `BaseConfiguration` so any configuration class should technically have this property.

File: message_tagging_service/tagging_service.py (hunk 1 above)
Comment (tkdchen) [reply message_tagging_service/tagging_service.py]: This my original solution. However, current version allows to print logs like this:

```
2019-01-11 10:01:41,354- message_tagging_service.tagging_service - INFO - [1] Checking rule definition: Building blocks                           
2019-01-11 10:01:41,357- message_tagging_service.tagging_service - INFO - [1] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,357- message_tagging_service.tagging_service - INFO - [2] Checking rule definition: Modular Python for Fedora                 
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [2] Rule definition: Matched.                                           
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [3] Checking rule definition: Modular updates for Fedora                
2019-01-11 10:01:41,358- message_tagging_service.tagging_service - INFO - [3] Rule definition: Matched.                                           
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [4] Checking rule definition: EPEL updates                              
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [4] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [5] Checking rule definition: Scratch builds                            
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [5] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,359- message_tagging_service.tagging_service - INFO - [6] Checking rule definition: Development builds                        
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [6] Rule definition: Not Matched.                                       
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [7] Checking rule definition: Fallback                                  
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - [7] Rule definition: Matched.                                           
2019-01-11 10:01:41,360- message_tagging_service.tagging_service - INFO - Tag build mariadb-10.4-20190106153700.a5b0195c with tag(s) f30-modular-up
dates-candidate, f30-modular-updates-candidate, modular-fallback-tag
```

I found the `[N]` part would make it easier to read and follow the logs.

File: message_tagging_service/__init__.py (hunk 2 above)
Comment (tkdchen) [reply message_tagging_service/__init__.py]: Yeah, it is not needed. :) Thanks.

File: tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+class TestRetrieveModulemdContent(object):
+    """Test utils.retrieve_modulemd_content"""
+
+    @patch.object(utils.mts_conf, 'mbs_api_url', new='https://mbs.local/')
+    @patch('requests.get')
+    def test_retrieve_the_content(self, get):
+        get.return_value = Mock(status_code=200)
+        get.return_value.json.return_value = {
+            'modulemd': 'modulemd content'
```
Comment (mvalik) [inline tests/test_utils.py]: The better practice is to set this to assign this string to some variable so if you change it in one place you don't have to change it in other.

File: fedmsg.d/logging.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Setup fedmsg logging.
+# See the following for constraints on this format https://bit.ly/Xn1WDn
+bare_format = "[%(asctime)s][%(name)10s %(levelname)7s] %(message)s"
+
+config = dict(
```
[the remark this answers is not in the document]
Comment (tkdchen) [reply fedmsg.d/logging.py]: Good point.

File: tests/test_utils.py (hunk 3 above)
Comment (tkdchen) [reply tests/test_utils.py]: Fixed.

File: message_tagging_service/tagging_service.py [hunk 5]
```diff
@@ -313,9 +311,8 @@ def handle(rule_defs, event_msg):
     nsvc = f"{this_name}-{this_stream}-{this_version}-{this_context}"
 
     try:
-        modulemd = yaml.safe_load(
-            retrieve_modulemd_content(
-                this_name, this_stream, this_version, this_context))
+        modulemd = retrieve_modulemd_content(event_msg['id'])
```
[the remark this answers is not in the document]
Comment (tkdchen) [reply message_tagging_service/tagging_service.py]: Fixed.

**Comment by tkdchen** [comment]
Rebased and fixed conflicts.

**Pull request merged**